# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContextEAGL.h
# Commit: dd00e2da3a0f
# Full Hash: dd00e2da3a0f11a7cc0af9e5ece21a339f4393eb
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 69cc1de6f754 -r dd00e2da3a0f gfx/gl/GLContextEAGL.h
--- a/gfx/gl/GLContextEAGL.h	Wed Jun 10 21:57:19 2020 +0000
+++ b/gfx/gl/GLContextEAGL.h	Wed Jun 10 22:21:02 2020 +0000
@@ -22,9 +22,8 @@
 
  public:
   MOZ_DECLARE_REFCOUNTED_VIRTUAL_TYPENAME(GLContextEAGL, override)
-  GLContextEAGL(CreateContextFlags flags, const SurfaceCaps& caps,
-                EAGLContext* context, GLContext* sharedContext,
-                bool isOffscreen, ContextProfile profile);
+  GLContextEAGL(const GLContextDesc&, EAGLContext* context,
+                GLContext* sharedContext, ContextProfile profile);
 
   ~GLContextEAGL();
 